request-example.go

```diff
@@ -151,6 +151,16 @@ func (fs *root) Filelist(r *Request) (ListerAt, error) {
 			return nil, err
 		}
 		return listerat([]os.FileInfo{file}), nil
+	case "Realpath":
+		fp := r.Filepath
+		if !filepath.IsAbs(fp){
+			fp = "/" + fp
+		}
```

Comment:
This needs a cleanPath() call here to canonicalize the path. Otherwise you'll end up with things like "/." as the path, which would fail to be found.

Reply:
It is only an example, but I agree that can be improved with `filepath.Clean`.